src/main/java/mate/jdbc/service/ManufacturerService.java

```diff
@@ -0,0 +1,17 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Manufacturer;
+
+public interface ManufacturerService {
+    Manufacturer create(Manufacturer manufacturer);
+
+    Optional<Manufacturer> get(Long id);
```

Comment:
The service must return an object of the class
```suggestion
    Manufacturer get(Long id);
```